crowdsourcehinter/static/js/src/crowdsourcehinter.js

```diff
@@ -0,0 +1,305 @@
+function CrowdsourceHinter(runtime, element, data){
+
+    var executeHinter = true;
+    var isShowingHintFeedback = false;
+    var voted = false;
```

Comment:
I would actually also do the same here. Highlight the ones where students voted, and ask about it.